pkg/web/api/v1/user.go

```diff
@@ -30,6 +36,10 @@ type UserPassword struct {
 	SendEmail bool   `json:"send_email"`
 }
 
+type UserQuery struct {
```

Comment:
The user query was added so that the FE may make a GET request to display the change password modal. We needed the ability to swap the modal into the `Users Management` page while also being able to access the ID of the user whose password will be changed. Instead of creating a separate route, the query was added similar to what was done in the `TransactionDetail` handler.